### ISSUE TITLE
Error: command 'cl.exe' failed: No such file or directory

### ISSUE THREAD

**Issue by geofbaum** [bcdev/jpy#116]
Trying to build the wheel file so that I can install the ESA Snappy module into Anaconda. Cloned the repo and went to execute the "python setup.py bdist_wheel" command and I got the error regarding the cl.exe file. I believe this is in regard to the Microsoft Visual Studio, and I have it set to look at the C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\Tools\ as is I believe suggested by the documents. I also have the JAVA_HOME set to C:\Program Files\Java\jdk1.8.0_151\ which I believe is right too. I'm not sure exactly where to go from here with this error. Am I missing something in the process?

I'm running this on Win10 64-bit, with Python 3.6 but can downgrade to 3.4 if absolutely necessary and if that's the issue but I don't think this particular issue is caused by that. Any help would be appreciated!

**Comment by geofbaum** [comment]
Also in case this helps anyone here is the full printout from the command prompt.

C:\Users\Geof Baum\Downloads\jpy>python setup.py bdist_wheel
Note that you can use non-standard global option [--maven] to force a Java Maven build for the jpy Java API
running bdist_wheel
running build
running build_py
copying jpyutil.py -> build\lib.win-amd64-3.6
running build_ext
building 'jpy' extension
cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -DWIN32=1 -Isrc\main\c "-IC:\Program Files\Java\jdk1.8.0_131\include" "-IC:\Program Files\Java\jdk1.8.0_131\include\win32" -ID:\Anaconda3\include -ID:\Anaconda3\include /Tcsrc\main\c\jpy_module.c /Fobuild\temp.win-amd64-3.6\Release\src\main\c\jpy_module.obj
error: command 'cl.exe' failed: No such file or directory

**Comment by forman** [comment]
> I'm running this on Win10 64-bit, with Python 3.6 but can downgrade to 3.4

No, your problem seems to be a configuration issue with MS Visual C++.

> I have it set to look at the C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\Tools\

So, in your environment you have  `VS100COMNTOOLS=C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\Tools\`, right?

**Comment by geofbaum** [comment]
Yes, as far as I can tell I do not know if there should be a response from the prompt after typing in the SET followed by the variable like so:
> SET VS100COMNTOOLS=C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\Tools\
> SET JAVA_HOME=C:\Program Files\Java\jdk1.8.0_151\

I apologize if this is something stupid on my end that I'm doing wrong.
